allensdk/test/brain_observatory/sync_utilities/test_sync_utilities.py

```diff
@@ -54,3 +54,17 @@ def test_get_synchronized_frame_times(monkeypatch, mock_dataset_fixture,
 
     obtained = su.get_synchronized_frame_times("dummy_path", sync_line_label_keys)
```

Comment:
Can you please add a test case to this with an example of trimming not just the last value?

Given the simplicity of this method it's not likely this is an edge case, but just for good test practice.

Reply:
Yep, I just added a couple tests, including a case where the spike is the very first entry in the list. Running all tests again now!